src/Development/IDE/LSP/Outline.hs

```diff
@@ -205,6 +208,9 @@ documentSymbolForImport (L l ImportDecl { ideclName, ideclQualified, ideclImplic
 documentSymbolForImport (L _ XImportDecl {}) = Nothing
 #endif
 
+isDamlInternalImport :: ModuleName -> Bool
+isDamlInternalImport ideclName = "DA.Internal" `T.isPrefixOf` pprText ideclName
```

Comment:
should we limit this to `DA.Internal.Desugar`? I’m a bit worried we have some useful stuff in `DA.Internal` that we reexport from other non-internal modules.

Reply:
Maybe at least `DA.Internal.Desugar` and `DA.Internal.Record`.

Reply:
sounds good :+1: